packages/samples/vuejs/src/components/FacetManager.vue

```diff
@@ -0,0 +1,67 @@
+<template>
```

Comment:
Based on this component's name, I would expect it to take facets as a parameter.

Instead of naming it `FacetManager`, I think we should name it something like `Facets`.

Reply:
it complains when it's one word only, that's why I added the 'Manager'. But I'll change to `SearchPageFacets`